index.js

```
@@ -1212,6 +1235,55 @@ class ModbusRTU extends EventEmitter {
+        // sanity check
+        if (typeof address === "undefined") {
+            if (next) next(new BadAddressError());
+            return;
+        }
+        const code = 21;
+        if (!data || data.length % 2 !== 0) {
+            if (next) next(new Error("Data length must be an even number of bytes"));
+            return;
+        }
+        const codeLength = 10 + data.length;
+        const requestDataLength = 7 + data.length;
```

ESLint

[error] 'requestDataLength' is assigned a value but never used.

(no-unused-vars)